datahub-frontend/app/controllers/Application.java

```diff
@@ -54,19 +54,38 @@ public Application(HttpClient httpClient, Environment environment, @Nonnull Conf
   }
 
   /**
-   * Serves the build output index.html for any given path
+   * Gets the configured base path for DataHub.
+   *
+   * @return the normalized base path
+   */
+  @Nonnull
+  private String getBasePath() {
+    return BasePathUtils.normalizeBasePath(config.getString("datahub.basePath"));
```

Comment:
Why does this need to be a method called for every request? Can we just do this once in the constructor and then return the computed String?

Reply:
Yeah, not longer needed.